### ISSUE TITLE
Add Laravel 5.5 compatiblity

### ISSUE THREAD

**Issue by freekmurze** [themsaid/laravel-mail-preview#32]
When installed in a Laravel 5.5 app there's an exception being thrown:

```
Declaration of Themsaid\MailPreview\PreviewTransport::send(Swift_Mime_Message $message, &$failedRecipients = NULL) must be compatible with Swift_Transport::send(Swift_Mime_SimpleMessage $message, &$failedRecipients = NULL)
```

This should be fixed 😄

**Comment by legreco** [comment]
With Laravel 5.5 you can preview mail in the browser 
https://laravel.com/docs/5.5/mail#previewing-mailables-in-the-browser

**Comment by freekmurze** [comment]
Will be fixed when #33 is merged.
